ci/pipeline.ts

```diff
@@ -0,0 +1,24 @@
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { SnykMonitorAddOn } from '../dist';
+
+const account = process.env.CDK_DEFAULT_ACCOUNT!;
+const region = process.env.CDK_DEFAULT_REGION!;
+const integrationId = process.env.INTEGRATION_ID;
+
+const eksBuilder = blueprints.EksBlueprint.builder()
+    .account(account)
+    .region(region)
+    .addOns(new SnykMonitorAddOn({ integrationId: integrationId }));
+
+export const pipeline = blueprints.CodePipelineStack.builder()
+    .name("snyk-monitor-eks-blueprints-addon-pipeline")
+    .owner("snyk-partners")
+    .repository({
+        repoUrl: 'ssp-eks-extension',
+        credentialsSecretName: 'ssp-pipeline-github-token',
+        targetRevision: 'develop'
+    })
+    .stage({
+        id: 'us-east-1-snyk-monitor-eks-blueprints-addon-test',
```

Comment:
Should this be hard-coded in here? I'm always nervous when I see IDs in commits.

Reply:
I think it's OK, since it's only the name of a pipeline stage, and I doubt we'll need to change it.

Reply:
👍